content/articles/whois-privacy.markdown

```diff
@@ -14,93 +14,128 @@ categories:
 
 ---
 
-The [WHOIS privacy protection service](/articles/what-is-whois-privacy) keeps your contact information private for a domain. This service replaces the public information that you provide with an anonymous proxy identity in the public WHOIS database.
+The [WHOIS Privacy protection service](/articles/what-is-whois-privacy) keeps your contact information private for a domain. This service replaces the public information that you provide with an anonymous proxy identity in the public WHOIS database.
 
-You can purchase the WHOIS privacy protection for any of your domains at any time, as long as the domain is registered with us and the TLD supports the WHOIS privacy service. You can check if a TLD supports WHOIS privacy from the [list of supported TLDs](https://dnsimple.com/tlds).
+You can purchase the WHOIS Privacy protection for any of your domains at any time, as long as the domain is registered with us and the TLD supports the WHOIS Privacy service. You can check if a TLD supports WHOIS Privacy from the [list of supported TLDs](https://dnsimple.com/tlds).
 
 <note>
-For [the new plans](/articles/new-plans) the price for the WHOIS privacy service starts from **$2/year per domain** and it is based on the currently subscribed plan. For all the [legacy plans](/articles/new-plans/#newer-plans-vs-legacy-plans) the price hasn't changed and it is $8/year per domain.
+For [the new plans](/articles/new-plans) the price for the WHOIS Privacy service starts from **$2/year per domain** and it is based on the currently subscribed plan. For all the [legacy plans](/articles/new-plans/#newer-plans-vs-legacy-plans) the price hasn't changed and it is $8/year per domain.
 </note>
 
 
-## Enable WHOIS privacy
+## Enable WHOIS Privacy {#enable}
 
-You can enable the WHOIS privacy protection from your domain page or the domain list.
+You can enable the WHOIS Privacy protection from your domain page or the domain list.
 
 When you enable the service the first time, the service cost will be charged on your account. You can then enable and disable the service at any time, with no additional cost, for one year.
 
 <div class="section-steps" markdown="1">
-##### Enable WHOIS privacy from the domain list {#enable-whois-domain-list}
+##### Enable WHOIS Privacy from the domain list {#enable-whois-domain-list}
 
 1.  Log into DNSimple with your user credentials.
 1.  Click on your avatar on the top-right, and on the drop-down menu select the account.
-1.  On the top-nav menu click the <label>Domains</label> tab. Locate the relevant domain and look for the button with the *shield* icon. That button will indicate whether the WHOIS privacy protection service is enabled for the domain:
+1.  On the top-nav menu click the <label>Domains</label> tab. Locate the relevant domain and look for the button with the _shield_ icon. That button will indicate whether the WHOIS Privacy protection service is enabled for the domain:
```

Comment:
We no longer display a button with a shield. You see the shield icon if WHOIS privacy is enabled for the domain.